src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -357,10 +357,14 @@
 			"  # +------------------------------------------------------+ #",
 			"  ############################################################",
 			""),
+	GTOWN_SETTINGS_MENU_LISTING(
+			"global_town_settings.menu_page-inator",
+			"true",
+			"# Should we Show the Fancy Town and Nation Lists (GUI)? Will use default text page-inator if false."),
 	GTOWN_SETTINGS_FRIENDLY_FIRE(
 			"global_town_settings.friendly_fire",
 			"true",
-			"# can residents/Allies harm other residents when in an area with pvp enabled? Other than an Arena plot."),

```

Comment:
Funky Merge, Removed and Re-Added lines. In the file it still exists. Happened again.